recipes/trgt/post-link.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
You don't need that don't any longer. Please delete

Reply:
done